### PR TITLE
fix(reports): updates vars in stock exp. report

### PR DESCRIPTION
**Pull request by jniles**
This commit fixes the stock expiration report which no longer worked after the variable case change landed in the stock performance updates.

### CODE REVIEW

**Comment by jniles** [comment]
@mbayopanda can I get a review?

**Comment by jniles** [comment]
Allo @mbayopanda can I get a review?

File: server/controllers/stock/reports/stock/expiration_report.js [hunk 1]
```diff
@@ -39,8 +39,9 @@ async function stockExpirationReport(req, res, next) {
     delete options.label;
 
     // define month average and the algo to use
-    options.monthAverageConsumption = req.session.stock_settings.month_average_consumption;
-    options.averageConsumptionAlgo = req.session.stock_settings.average_consumption_algo;
+    // eslint-disable-next-line
+    const { month_average_consumption, average_consumption_algo } = req.session.stock_settings;
+    _.extend(options, { month_average_consumption, average_consumption_algo });
```
Comment (mbayopanda) [inline server/controllers/stock/reports/stock/expiration_report.js:44]: 👍

**Review by mbayopanda — APPROVED** [review]
LGTM 👍 

bors r+

**Pull request merged**